src/main/java/mate/academy/spring/config/DataInitializer.java

```diff
@@ -0,0 +1,44 @@
+package mate.academy.spring.config;
+
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.RoleName;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataInitializer {
+    private final RoleService roleService;
+    private final UserService userService;
+
+    @Autowired
```

Comment:
We do the same in all constructors of classes, where we need it: either we put the annotation or not